webapp/src/pages/Settings.tsx

```diff
@@ -1,5 +1,11 @@
-import { Close, Warning } from "@mui/icons-material";
 import {
+  AddCircleOutline,
```

Comment:
We normally use the filled variants of the icons. I think it would be best, especially to align with the `DeleteForever` icon which is filled.
```suggestion
  AddCircle,
```